content/pages/education/apply.md

```diff
@@ -56,10 +76,9 @@ If you need to make a change (for example, you’re moving to a new school), man
 <div class="usa-alert usa-alert-warning usa-content va-alert" markdown="1">
 	<div class="usa-alert-body">
 
-##### Forms 22-1990E, 22-1990N, 22-5490, and 22-5495
+##### Forms 22-1990N, 22-5490, and 22-5495
 
 You must apply for education benefits using eBenefits if you're:
-- A spouse or dependent
```

Comment:
The only thing I think we want to be careful about is that even though the 5490 is for "survivors", which is the bullet below, they may see themselves as "spouses and dependents" still and be confused about where to go if we remove that line. So maybe this is something both @J-Quag and @goldenmeanie can weigh in on; perhaps we need more detail here for now?